src/Logger/Logger.cc

```diff
@@ -109,7 +109,8 @@ void InitLogger(bool no_log_file, int verbosity, bool log_performance, bool log_
 }
 
 void LogReceivedEventType(const CARTA::EventType& event_type) {
-    if (log_protocol_messages) {
+    auto& settings = ProgramSettings::GetInstance();
+    if (settings.log_protocol_messages) {
```

Comment:
This function only uses that one boolean flag -- so the previous behaviour should be restored. That flag can be set inside the init function, as before.